woodwork/column_accessor.py

```diff
@@ -434,6 +484,59 @@ def box_plot_dict(
         """
         return _get_box_plot_info_for_column(
             self._series,
+            method="box_plot",
+            quantiles=quantiles,
+            include_indices_and_values=include_indices_and_values,
+            ignore_zeros=ignore_zeros,
+        )
+
+    @_check_column_schema
+    def medcouple_dict(
+        self,
+        quantiles: Optional[Dict[int, int]] = None,
+        include_indices_and_values: bool = True,
+        ignore_zeros: bool = False,
+    ):
+        """Gets the information necessary to create a box and whisker plot with outliers for a numeric column
+        using the Medcouple statistic method.
+
+        Args:
+            quantiles (dict[float -> float], optional): A dictionary containing the quantiles for the data
+                where the key indicates the quantile, and the value is the quantile's value for the data. If
+                no quantiles are provided, they will be computed from the data.
+            include_indices_and_values (bool, optional): Whether or not the lists containing individual
+                outlier values and their indices will be included in the returned dictionary.
+                Defaults to True.
+            ignore_zeros (bool): Whether to ignore 0 values (not NaN values) when calculating the box plot and outliers.
```

Comment:
this will probably eventually be a semantic tag on the column - just curious if we'd still want that to be an input or if we have access to that via the series. NAB

Reply:
Once it becomes a semantic tag on the column we should access it via the series, for now I'd prefer to keep it as an argument